### PR TITLE
Emotion setup

### LINKED ISSUE #1: Setup Emotion

### PR DESCRIPTION
**Pull request by axelekwall**
Emotion setup with theme and normalize.css. It requires the use of styled from the file `src/styles/styled.ts` rather than the default styled import for correct theme typing. The theme can be found in `src/styles/theme.ts`.

fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,30 +3,37 @@
...
 The `/src/pages` folder is a bit special, but fits well into the atomic design structure. Gatsby will create a new page for each file in this folder that exports a react component. This is how to create basic routing.
 
 Pages can also be generated programmatically.
 
 ### Styling
 
-We will use Emotion, more info soon...
+We will use Emotion for styling. This is a tools that works very much like styled components, but with a few nice improvements.
+
+There are great dics to read for more info [here](https://emotion.sh/docs/introduction).
```
Comment (johannesloor) [inline README.md]: Maybe it's supposed to be "docs"... ?

Comment (johannesloor) [reply README.md]: On line 38

Comment (axelekwall) [reply README.md]: Haha, yes it's supposed to be "docs" 😄

Comment (axelekwall) [inline README.md]:
```suggestion
There are great docs to read for more info [here](https://emotion.sh/docs/introduction).
```

**Pull request merged**